pycqed/instrument_drivers/meta_instrument/LutMans/mw_lutman.py

```diff
@@ -666,19 +660,15 @@ def _add_channel_params(self):
                 parameter_class=ManualParameter, initial_value=None,
                 vals=vals.Ints())
 
-    def load_waveform_realtime(self, wave_id,
-                               wf_nr: int = None,
-                               regenerate_waveforms: bool=True):
+    def load_waveform_onto_AWG_lookuptable(
```

Comment:
I see this method is added but no corresponding method is removed in the old version. Is there a reason to redefine this or maybe I'm just fooling myself.

Reply:
I was under the impression that `load_waveform_onto_AWG_lookuptable` was the 'standard' way of naming this kind of function within your framework. Also, you have the `load_waveforms_onto_AWG_lookuptable` for loading all waveforms, which has consistent naming with this method.